### PR TITLE
validation for user in section edit

### PR DESCRIPTION
**Pull request by ZulfanoRp**

### CODE REVIEW

File: public/admin/js/dataUser.js [hunk 1]
```diff
@@ -344,21 +344,46 @@ $(document).ready(function () {
...
+adminTab.addEventListener("click", () => {
+    localStorage.setItem("activeTab", "admin");
+});
+userTab.addEventListener("click", () => {
+    localStorage.setItem("activeTab", "user");
+});
+
+// Close alert create admin
 $("#closeAlertAdmin").click(function () {
     $("#alertAdmin").hide(300);
```
Comment (KrishnaHendra) [inline public/admin/js/dataUser.js:377]: interval show/hide e di sama ratano 500 ae kyok wong" biasae lek gak 1000 cek ngga auto show kyok bug

**Review by KrishnaHendra — CHANGES_REQUESTED** [review]
laine wes ok jos

File: public/admin/js/dataUser.js [hunk 2]
```diff
@@ -344,21 +344,46 @@ $(document).ready(function () {
...
 $(".custom-file-input").on("change", function () {
     var fileName = $(this).val().split("\\").pop();
     $(this).siblings(".custom-file-label").addClass("selected").html(fileName);
 });
 
+// Localstorage for tab-pane admin and user
+let adminTab = document.querySelector("#adminTab");
+let userTab = document.querySelector("#userTab");
+
+let activeTab = localStorage.getItem("activeTab");
```
[the remark this answers is not in the document]
Comment (ZulfanoRp) [reply public/admin/js/dataUser.js:357]: hmm okay

File: public/admin/js/dataUser.js (hunk 1 above)
Comment (ZulfanoRp) [reply public/admin/js/dataUser.js:377]: oh oke oke

**Pull request merged**